testsuite/tests/backtrace/pr2195.run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export OCAMLRUNPARAM=b
```

Comment:
See comment above - this hook is only used to ensure that the test is run with `OCAMLRUNPARAM=b`